### PR TITLE
Foreign section updates

### PR DESCRIPTION
**Pull request by floresj**
Note this has updates merged in from the `adjust-help-icon-alignment` branch to account for the help updates. Don't merge until `adjust-help-icon-alignment` is in.

- Updating rules for date controls to allow future dates on some fields
- Fixed currency not validating

### CODE REVIEW

**Comment by bmallred** [comment]
@floresj is this ready for review?

**Comment by floresj** [comment]
@bmallred Almost. One more update to do.

**Comment by floresj** [comment]
ooook now I think it's good to go.

File: src/config/locales/en.js [hunk 1]
```diff
@@ -4849,7 +4859,9 @@ const en = {
           title: 'Have you, your spouse or legally recognized civil union/domestic partner, cohabitant, or dependent children EVER had any foreign financial interests in which you or they have direct control or direct ownership?'
         },
         para: {
-          intro: 'Foreign financial interest examples:  stocks, property, investments, bank accounts, ownership of corporate entities, corporate interests or exchange traded funds (ETFs) held in specific geographical or economic sectors.\n\n**Exclude financial interests in companies or diversified mutual funds or diversified ETFs that are publicly traded on a U.S. exchange.**'
+          intro: 'Foreign financial interest examples:  stocks, property, investments, bank accounts, ownership of corporate entities, corporate interests or exchange traded funds (ETFs) held in specific geographical or economic sectors.\n\n**Exclude financial interests in companies or diversified mutual funds or diversified ETFs that are publicly traded on a U.S. exchange.**',
```
Comment (bmallred) [inline src/config/locales/en.js]: We could break this in to an array if you are calling it with `i18n.m()`

example:

```
{
  intro: [
    'Foreign financial interests examples: stocks, ...',
    '**Exclude financial interests in companies...**'
  ]
}
```

Comment (floresj) [inline src/config/locales/en.js]: Excellent I'll arrify it.

**Pull request merged**